atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -126,6 +126,8 @@ public int execute(AtlasDbServices fromServices, AtlasDbServices toServices) {
         }
         if (migrate) {
             migrator.migrate();
+
+            migrator.cleanup();
```

Comment:
Before this change, was it possible to resume a migration that was interrupted?

Also: the AtlasDB migration CLI is required to be run offline, but I wonder if before this change you could do most of the migration online by running the migration to 'completion' while the database was in use, then shutting everything down for a short downtime and then re-running 'migration' to complete the migration.